api/src/controllers/RequestAppAccessController.js

```diff
@@ -0,0 +1,54 @@
+import fs from 'fs';
```

Comment:
Would prefer that we moved towards individual files for each request handler, but that change hasn't been merged into master so don't worry about it here, just something to bear in mind in the future once [LL-228](https://learningpool.atlassian.net/browse/LL-228) is merged.

Reply:
ok thanks